src/serde_helpers.rs

```diff
@@ -21,13 +21,18 @@ pub use iso_string_as_bson_datetime::{
     serialize as serialize_iso_string_as_bson_datetime,
 };
 pub use timestamp_as_u32::{
-    deserialize as deserialize_timestamp_from_u32, serialize as serialize_timestamp_as_u32,
+    deserialize as deserialize_timestamp_from_u32,
+    serialize as serialize_timestamp_as_u32,
 };
+pub use u32_as_f64::{deserialize as deserialize_u32_from_f64, serialize as serialize_u32_as_f64};
```

Comment:
I opted to only include helpers for u32 and u64 since I doubt any of the smaller oners (`u8` or `u16`) are used very often. If someone asks for them we can add them back.

Reply:
sounds reasonable, fine by me